searx/templates/simple/macros.html

```diff
@@ -21,9 +21,29 @@
 {% macro result_header(result, favicons, image_proxify) -%}
 <article class="result {% if result['template'] %}result-{{ result.template|replace('.html', '') }}{% else %}result-default{% endif %} {% if result['category'] %}category-{{ result['category'] }}{% endif %}{% for e in result.engines %} {{ e }}{% endfor %}">
   {{- result_open_link(result.url, "url_wrapper") -}}
+  {% if not rtl %}
```

Comment:
I don't really like the duplicated code for rtl/ltr. Can't we just do that via CSS by changing the flex direction?